hadoop-tools/hadoop-aws/src/test/resources/core-site.xml

```diff
@@ -178,6 +178,12 @@
     <value>true</value>
   </property>
 
+  <!-- Enable IOStatisticsContext support for Thread level. -->
```

Comment:
mixed feelings here. hive team and others who serialize configs don't like us overfilling the xml file with defaults

Reply:
They might not have issue with test resource xml changes right?

Reply:
no issues there at all

Reply:
lets go with it